packages/ui-design-system/src/Modal/Modal.tsx

```diff
@@ -5,7 +5,7 @@ import clsx from 'clsx';
 import { forwardRef } from 'react';
 
 const modalContentClassnames = cva(
-  'bg-grey-00 fixed left-1/2 top-1/2 flex h-fit w-full -translate-x-1/2 -translate-y-1/2 flex-col rounded-lg drop-shadow-xl overflow-hidden',
+  'bg-grey-00 fixed left-1/2 top-[10vh] flex h-fit w-full -translate-x-1/2 flex-col rounded-lg drop-shadow-xl overflow-hidden',
```

Comment:
Required to fix a weird bug, with CSS translate, text is blured when a scroll indicator appear (not sure if it's visible on the picture, but for me it was clearly visible on my screen)

![image](https://github.com/user-attachments/assets/6265b7f8-e828-4e9a-8325-c1ef2c9d5195)